Admin/AdminBuilder.php

```diff
@@ -22,7 +22,7 @@
 /**
  * @author Christian Gripp <mail@core23.de>
  */
-final class AdminBuilder implements FormBuilderInterface, DatagridBuilderInterface
+final class AdminBuilder implements AdminBuilderInterface
```

Comment:
What if I need to mock AdminBuilder, is there a way to generate a mock extending two interfaces?

IMO this is similar to: https://github.com/sonata-project/SonataMediaBundle/pull/1174/files#diff-506bd842fd8fd25296a34c132e9479d8L22